apps/kontrol-bot/events/moderationGunaydinChannel.ts

```diff
@@ -0,0 +1,24 @@
+import { Events, Message } from "discord.js";
+
+export default {
+  name: Events.MessageCreate,
+  async execute(message: Message) {
+    if (message.author.bot) return;
+
+    // If the message is sent to "Günaydın" text channel
+    if (message.channel.id === "1158288028511522877") {
+      // Check if the message includes "günaydın" or similar
+      // or a GIF or the 🌞 emoji.
+      const morningPattern = /(g+(u|ü)+n+a+y+d+(ı|i)+n+)|(gif)|(🌞)/i;
```

Comment:
We couldn't check if there is a "sun" string in the emojis (eg: :sun_with_face:), since message.content directly comes as an emoji. Should we include all related possible emojis with "|", or should we look more into it?

Reply:
this looks fine